public/stylesheets/site/2.0/08-actions.css

```diff
@@ -50,6 +50,39 @@ button {
   box-sizing: content-box;
 }
 
+.actions form.button_to {
+  line-height: 1.5
+}
```

Comment:
Could you tell me what browser or page this is needed for? It wasn't immediately clear to me from the code, so I took it out while I was checking over all the pages and couldn't find any obvious issues.

Reply:
Removed.